pkg/kubelet/images/image_gc_manager.go

```diff
@@ -54,6 +54,9 @@ type ImageGCManager interface {
 	// Start async garbage collection of images.
 	Start()
 
+	// Stop async garbage collection of images.
```

Comment:
No one calls stop, why would you need this?

If you plan on adding stop, do so in this commit.

/hold

Reply:
I added the call to Stop()

Please take another look.